c2corg_ui/templates/i18n.html

```diff
@@ -88,24 +83,20 @@
 <span translate>Editing a route</span>
 <span translate>Creating a waypoint</span>
 <span translate>Editing a waypoint</span> 
+<span translate>Creating an outing</span>
+<span translate>Editing an outing</span>
 
 ## home
 <span translate>Home</span>
 
 ## waypoint attributes
-<span translate>rain_proof</span>
 <span translate>very_safe</span>
 <span translate>exposed</span>
-<span translate>rock_types</span>
-<span translate>best_periods</span>
-<span translate>climbing_styles</span>
```

Comment:
Yes, those strings are already available in HTML templates, thus are already extracted from those templates. We don't need them here.